test/system/500-networking.bats

```diff
@@ -676,12 +676,12 @@ EOF
 
 @test "podman run port forward range" {
     for netmode in bridge slirp4netns:port_handler=slirp4netns slirp4netns:port_handler=rootlesskit; do
-        local port=$(random_free_port)
-        local end_port=$(( $port + 2 ))
-        local range="$port-$end_port:$port-$end_port"
+        local range=$(random_free_port_range)
+        local port="${test%-*}"
+        local end_port="${test#-*}"
```

Comment:
Just a glance, not a real review, but: did you consider having the helper function return only the starting port, and continuing to do the math here? `port+2` seems more readable than `%-*` and `#-*`.

Reply:
I think it makes sense for the function to return it as `port1-port2`, but I can change it if you prefer that.